Techup 24 Banxa and Github Issue And Previous Commits Review

TechUp-24 opened the pull request.

itsafuu committed:
Automatically detect platform. Hardcode phase 3.1 for now

itsafuu committed:
Made quicktype gen check if quicktype is installed. Added run.sh to run without making a build specifically

itsafuu committed:
Added phase 3.1 args to bat file.

itsafuu committed:
Large readme update to simplify how to build/run this project.

itsafuu committed:
install flutter scripts

itsafuu committed:
Flutter install script instructions

itsafuu committed:
Needed leading space for this to work on windows for some reason

itsafuu committed:
Launch config for vscode

itsafuu committed:
Added VSCode instructions

EduMenges committed:
Added escrow release draft

Ammaryasirkhan committed:
complete sandbox buy-flow

Ammaryasirkhan committed:
My sandbox buy-flow changes

Ammaryasirkhan committed:
WIP: save work before switching to develop

Ammaryasirkhan committed:
Sand Box Feature

Ammaryasirkhan committed:
Banxa QR checkout, redirectUrl handling, deep link groundwork

Ammaryasirkhan committed:
Local backup: Banxa QR + redirect flow + submodule pointer

Ammaryasirkhan committed:
backup

Ammaryasirkhan committed:
Save current work before switching branch

Ammaryasirkhan committed:
added KYC

Ammaryasirkhan committed:
added some new fixes

Ammaryasirkhan committed:
dark theme added

Ammaryasirkhan committed:
added resposnsiveness fixes

Ammaryasirkhan committed:
added new banxa gen-code.sh

Ammaryasirkhan committed:
Saving progress before merging into banxa-sandbox

Ammaryasirkhan committed:
Committing changes before merging into develop

TechUp-24 committed:
Temp commit: Save pubspec changes before rebase

TechUp-24 committed:
Merge feature/banxa-pre-deeplink into develop

TechUp-24 committed:
Fix: Updated submodule URLs to GeniusVentures versions

TechUp-24 committed:
Merge: upstream/develop into develop to align with GeniusVentures

TechUp-24 committed:
Fix: Updated submodule URLs to relative paths

TechUp-24 committed:
Techup Banxa and Github Issues

TechUp-24 committed:
Remove unwanted build files from tracking

TechUp-24 committed:
Remove generated and user-specific files from tracking

TechUp-24 committed:
Merge upstream/develop into techup_24 and resolve conflicts

itsafuu commented:
I wanted you to remove the cmake generated files from ios/native and macos/native, not the original files that needed to be there.

TechUp-24 committed:
Fix: Ignore only CMake-generated files in ios/native and macos/native

TechUp-24 committed:
Fix: Ignore only CMake-generated files in ios/native and macos/native

TechUp-24 commented:
I've added back the missing necessary files in both ios/native and macos/native) and adjusted .gitignore to only exclude generated files.

Apologies for the earlier oversight — thank you for the patience!

TechUp-24 committed:
fix: update intl dependency for device_preview compatibility

TechUp-24 committed:
flutter upgrade for the dependecies

TechUp-24 committed:
dependecies fixed

TechUp-24 committed:
dependecies fixes

TechUp-24 committed:
compile sdk 36

TechUp-24 committed:
fl_chart change to 1.1.1

TechUp-24 committed:
chnaged back to 1.1.0

TechUp-24 committed:
new

TechUp-24 committed:
fix: Remove local signing profile for CI/CD compatibility

TechUp-24 committed:
changed the fl_chart
